.github/workflows/ci.yml

```diff
@@ -38,7 +35,8 @@ jobs:
                   echo "## Lint result" >> $GITHUB_STEP_SUMMARY
                   echo "✅ Passed" >> $GITHUB_STEP_SUMMARY
 
-    build:
+    proposal-simulator-test:
+        needs: ["lint"]
```

Comment:
I don't think it's needs lint job

Reply:
I kept it same as before. I think tests should run only if lint passes.